antsibull/cli/antsibull_docs.py

```diff
@@ -50,6 +51,9 @@
 def _normalize_docs_options(args: argparse.Namespace) -> None:
     args.dest_dir = os.path.abspath(os.path.realpath(args.dest_dir))
 
+    if args.command == 'sphinx-init':
+        return
```

Comment:
Maybe the answer to the makedirs exists_ok=True issue is to do something differently in this section?  Are you skipping this section because dest_dir may not already exist?  Maybe we should look into how to make it secure for us to create dest_dir here.

Reply:
Yes, that's exactly the reason. I guess what we could do is figure out the first parent of the specified directory that exists (by iterating over `os.path.dirname`), and make sure that one satisfies the conditions (instead of checking `dest_dir` directly).

Reply:
For now I removed this line (c361cda84d9a533af973909e40f80b6403b84e8d), so that dest_dir is expected to exist and have the (currently) 'correct' permissions as well. We can remove that restriction for all docs subcommands in one go in another PR (and improve validation) as discussed on IRC.